src/main/java/nl/tudelft/lifetiles/Launcher.java

```diff
@@ -50,7 +51,7 @@
      * @param args
      *            The program arguments
      */
-    public static void main(final String[] args) {
+    public static void main(final String... args) {
```

Comment:
I'm sorry, this technically should be in this pull-request, but I got annoyed by PMD.